kai/routes/change_model.py

```diff
@@ -0,0 +1,8 @@
+from aiohttp.web_request import Request
+
+from kai.routes.util import to_route
+
+
+@to_route("post", "/change_model")
+async def post_change_model(request: Request):
```

Comment:
what is this for?

Reply:
Originally had this route if the user wanted to change the model via a POST request, but it's dead code as of now. Removing.